bundles/org.openhab.binding.miio/src/main/resources/OH-INF/thing/vacuumThing.xml

```diff
@@ -299,6 +315,18 @@
 		<item-type>String</item-type>
 		<label>Vacuum Room [room#]</label>
 	</channel-type>
+	<channel-type id="collectdust" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Start collect the dust at the station</label>
```

Comment:
See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions:
```suggestion
		<label>Start Collect the Dust at the Station</label>
```

however, it should also be kept short (2-3 words with up to 25 chars). Perhaps you could shorten the label and provide the current label as `description` instead?